src/main/java/uk/gov/hmcts/sptribs/document/dto/CicBundleDocumentDTO.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.sptribs.document.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.sptribs.document.model.CICDocument;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
+@Slf4j
+public class CicBundleDocumentDTO {
+
+    private String name;
+    private String description;
+    private int sortIndex;
+    private CICDocument sourceDocument;
+
```

Comment:
Can use Lombok @AllArgsConstructor and @NoArgsConstructor here?

Reply:
Review comment fixed.